### PR TITLE
Revised Pull Request

### PR DESCRIPTION
**Pull request by neelbagora**
Revised previous pull request to contain more descriptive variable names and better formatting in script.js.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -11,18 +11,69 @@
...
+    }
+}
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addClass(element, name) {
```
Comment (avenkat-google) [inline portfolio/src/main/webapp/script.js:28]: Can you also update the names of addClass and removeClass to make their behaviors more explicit (addClass doesn't communicate that this function shows the element).
Also do we need the name parameter? Seems like "show" is just added or removed, so the functions can probably be simplified.

Comment (neelbagora) [reply portfolio/src/main/webapp/script.js]: I added in the second parameter to make it accommodating of any future classes I might want to animate into the html.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -11,18 +11,69 @@
...
+function updateGalleryText(elementName) {
+    const airplane = "These are photos I have taken from the flight simulator, X-Plane 11. In my free time, I love to simulate air traffic communication during my flights so I use the VATSIM network. The VATSIM network is a plugin used with X-Plane that allows me to communicate with real people (acting as air traffic controllers) and fly under the supervision of air traffic controllers. The callsigns mentioned below are callsigns used on the VATSIM online air traffic network.";
+    const projects = "These are pictures of the projects I have worked on throughout the years. I am mainly experienced in Java, C, C++, and Python. A lot of my projects were built when I wanted to learn something new, for example, the currency converter gave me a chance to learn both Kotlin and API calls in Kotlin/Java. The MIDI music program was a way for me to learn how binary files work in real world context.";
+    const miscellaneous = "These are pictures that I thought were cool but could not categorize.";
+    const showAll = "These are all the pictures that I have posted on this website, pictures can be filtered using the menu bar above.";
```
[the remark this answers is not in the document]
Comment (neelbagora) [reply portfolio/src/main/webapp/script.js]: Just fixed this.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (avenkat-google) [reply portfolio/src/main/webapp/script.js]: Oh I see. In that case for this PR, we can keep the funcitonality as is, but can we change the name to AddElementClass or RemoveElementClass?

Comment (neelbagora) [reply portfolio/src/main/webapp/script.js]: Yeah, just added that right now.

**Pull request merged**